client/src/index.js

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App';
-import * as serviceWorker from './serviceWorker';
+import { Route, BrowserRouter as Router } from 'react-router-dom'
+import Profile from './Components/Pages/Profile';
+import Experts from './Components/Pages/Experts';
+import Learners from './Components/Pages/Learners';
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
-);
+const routing = (
+  <Router>
```

Comment:
I have a feeling this would work better to put inside app as we will be adding all the auth providers in this file so it might be clearer to separate.